shell/platform/windows/win32_task_runner.h

```diff
@@ -15,14 +15,15 @@
 #include <thread>
 
 #include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/windows/task_runner.h"
 
 namespace flutter {
 
 typedef uint64_t (*CurrentTimeProc)();
 // A custom task runner that integrates with user32 GetMessage semantics so that
 // host app can own its own message loop and flutter still gets to process
 // tasks on a timely basis.
-class Win32TaskRunner {
+class Win32TaskRunner : public TaskRunner {
```

Comment:
We should really be calling these classes things like `TaskRunnerWin32`/`TaskRunnerWinRt` rather than the reverse. Let's fix that for this class in this PR (since it's adding a new file with the naming we should move away from), and I can do a follow-up to clean up the others after it lands.